documentation/airport-guides/lszh.md

```diff
@@ -0,0 +1,63 @@
+
+#### LSZR
+Flugplatz St. Gallen-Altenrhein (St. Gallen–Altenrhein Airport)  
+26,382 annual operations (2016)  
+40nm to the east of LSZH  
+
+St. Gallen–Altenrhein Airport is a small airport in Altenrhein in the Canton of St. Gallen, Switzerland near Lake Constance. It is also known under its trading name People's Business Airport and is the home base for People's Viennaline. 
+
+
+#### EDNY
+Flughafen Friedrichshafen (Friedrichshafen Airport)  
+42,374 annual operations (2016)  
+43nm to the east of LSZH  
+
+Friedrichshafen Airport is a minor international airport 1.9 miles (3 km) north of Friedrichshafen, Germany, on the banks of Lake Constance (German: Bodensee). It is the third biggest airport in the German state of Baden-Württemberg after Stuttgart and Karlsruhe/Baden-Baden and served 559,985 passengers in 2015. Friedrichshafen features flights to European metropolitan and leisure destinations. Due to its proximity to the Austrian Alps it is also heavily used during the winter by skiing tourists. 
+
+
+#### LSME
+Militärflugplatz Emmen (Emmen Military Air Base)  
+25nm to the south of LSZH  
+
+The airfield of Emmen is used primarily for military use and is operated by the Swiss Air Force as training and alternate airfield for jet and propeller aircraft as well as the ADS95 RUAG Ranger UAV. This means that no jet aircraft squadron is stationed in Emmen, but the airfield is still used as a training and alternate air base. The air base is, at both end of the runway, equipped with retractable arresting gear devices (used by the F/A-18 and in case of a problem by the F-5). Next to the aerodrome serves the following military uses
+
+
+#### LSMD
+Militärflugplatz Dübendorf (Dübendorf Military Air Base)  
+5nm to the south of LSZH  
+
+Today Dübendorf AFB is the home of Lufttransportstaffel 3 and Lufttransportstaffel 4 (Air Transport squadron 3 and 4) with Eurocopter EC635, Eurocopter AS332 Super Puma, Eurocopter AS532 Cougar, De Havilland Canada Twin Otter, Beechcraft 1900 and Beechcraft Super King Air. Dübendorf is, together with Locarno AFB, the home base of the PC-7 Team and the Superpuma/cougar Displayteam. Other Swiss military aircraft such as the Pilatus PC-6 Turbo-Porter and Pilatus PC-9 are often seen in Dübendorf.
+
+### Additional Resources
+VATSIM Charts (http://www.vacc.ch/en/airports_and_charts/)  
+Jeppesen Charts (http://vau.aero/navdb/chart/LSZH.pdf)  
```

Comment:
Would not recommend putting Jeppensen charts here; the charts from VATSIM will be sufficient